config.json

```diff
@@ -866,9 +866,11 @@
       "slug": "connect",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 9,
       "topics": [
-
+        "Control-flow (loops)",
+        "Games",
+        "Graphs"
```

Comment:
"Arrays" applies I think.
Also "Searching", since the program searches for a winner if one exists.

I didn't think that Graphs applied, but maybe it is a valid perspective.